packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-env browser */
+
+/* global Webex */
+
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+/* eslint-disable no-global-assign */
+/* eslint-disable no-multi-assign */
+
+// Globals
+let webex;
+
+// Collapsible Setup
+const collapsibles = document.querySelectorAll('.collapsible');
+
+function collapse() {
+  const prevState = this.getAttribute('data-collapsed');
+  const newState = prevState === 'true' ? 'false' : 'true';
+
+  this.setAttribute('data-collapsed', newState);
+  this.nextElementSibling.setAttribute('data-collapsed', newState);
+}
+
+collapsibles.forEach(
+  (collapsible) => {
+    collapsible.addEventListener('click', collapse);
+  }
+);
+
+// Authentication Section
+
+const tokenElm = document.querySelector('#access-token');
+const saveElm = document.querySelector('#access-token-save');
+const authStatusElm = document.querySelector('#access-token-status');
+const registerElm = document.querySelector('#registration-register');
+const unregisterElm = document.querySelector('#registration-unregister');
+const registrationStatusElm = document.querySelector('#registration-status');
+
+function saveCredentials() {
+  console.log('Authentication#saveCredentials()');
```

Comment:
I'm worried that we're creating too much of an abstraction here. `saveCredentials` isn't an actual function a user could call and might cause confusion. Would prefer this to directly refer to the `Webex.init` function we call.

Reply:
Updated to `initWebex()`